test/functional/rpc_blockchain.py

```diff
@@ -163,7 +163,7 @@ def _test_getblockchaininfo(self):
                     'status': 'started',
                     'bit': 28,
                     'start_time': 0,
-                    'timeout': 999999999999,  # testdummy does not have a timeout so is set to the max int64 value
```

Comment:
I think should update also these functional tests for consistancy:

test/functional/feature_new_quorum_type_activation.py:
`self.extra_args = [["-vbparams=testdummy:0:999999999999:10:8:6:5"]]`
test/functional/feature_llmq_data_recovery.py:
`-vbparams=testdummy:0:999999999999:10:8:6:5`

Reply:
as discussed with @kwvg indeed bitcoin didn't care to update 999999999999 for segwit deployment in python code.

Make sense, because typing `9223372036854775807` in args not much convenient and looks pretty much as magic number.

Let's keep there `testdummy:0:999999999999:10:8:6:5` as it is